### ISSUE TITLE
One or more js files not found

### ISSUE THREAD

**Issue by lukedukeus** [turbobabr/easyeda-quick-align-extension#1]
I get the error "One or more js files not found" when trying to add the extension into easyeda, I'm assuming I just add all the contents of the `src` directory?

**Comment by l0rdl4nx** [comment]
I have the same problem... Did you find a workaround yet?

**Comment by lukedukeus** [comment]
No, I found that the extension still works as expected, just shows the error at startup
